arch/risc-v/src/k230/Kconfig

```diff
@@ -24,20 +24,41 @@ config K230_RPTUN_SHM_SIZE
 
 config K230_RPTUN_IPI
 	bool "use IPI for RPTUN"
-	default n
-	depends on K230_IPI
+	default K230_IPI
 	---help---
 		Use Inter-Processor interruption to improve efficiency.
 		Not ready yet.
 
-
 config K230_RPTUN_WORK_DELAY
 	int "RPTUN work delay"
-	default 50
-	depends on !K230_RPTUN_IPI
+	default 100
 	---help---
-		 Ticks for polling RPTUN. Polling doesn't need IPI.
+		Ticks for polling RPTUN. Polling doesn't depend on IPI and IPI
```

Comment:
why not aleady enable IPI and remove the polling code directly to simplify the code logic?